pom.xml

```diff
@@ -25,7 +25,7 @@
         <changelist>-SNAPSHOT</changelist>
         <hpi.compatibleSinceVersion>2.2.0</hpi.compatibleSinceVersion>
         <java.level>8</java.level>
-        <jenkins.version>2.270</jenkins.version>
+        <jenkins.version>2.274</jenkins.version>
```

Comment:
Do we really want to bump this? Someone running 2.270 is probably better off running a yet-to-be-released version of the plugin that includes #360 than the an older release even though there would be some additional issues until they update Jenkins past 2.274.

Is there any reason (e.g. missing API) we shouldn't bump the version back down to 2.264 so that everyone running tables to divs at least has access to #360?

Reply:
Hmm the plan was to have this working in hpi:run I would like 2.271 really

Reply:
I guess it is reasonable to expect anyone running weekly releases to always be running the latest weekly unless there is some serious issue, so from that perspective, until the next LTS is released it doesn't really matter too much what we use for the minimum version here, and 2.271+ definitely would be simplest for testing purposes. @bitwiseman what do you think?

Reply:
@dwnusbaum @timja
I think we should be conservative with this.  Pick the lowest number that includes what we need. We really want to make sure we pick a version that makes into the the next LTS.